src/Rings/mpolyquo-localizations.jl

```diff
@@ -1054,8 +1056,7 @@ end
 # uncommon implementation of the numerator and denominator methods
 function (f::MPolyQuoLocalizedRingHom)(a::AbsLocalizedRingElem)
   parent(a) === domain(f) || return f(domain(f)(a))
-  isone(lifted_denominator(a)) && return codomain(f)(restricted_map(f)(lifted_numerator(a)))
-  return codomain(f)(restricted_map(f)(lifted_numerator(a)))*inv(codomain(f)(restricted_map(f)(lifted_denominator(a))))
+  return codomain(f)(restricted_map(f)(lifted_numerator(a)), check=false)*inv(codomain(f)(restricted_map(f)(lifted_denominator(a)), check=false))
```

Comment:
Why take this back?

Reply:
No clue, I did not know what was the old version and what the new one.
So I will revert to the old version.